SBAT.md

```diff
@@ -73,20 +73,22 @@ number are required to include fixes for any public or pre-disclosed
 issue required for that generation. Additionally, in the event that a
 bypass only manifests in a specific product's component, vendors may
 ask for a product-specific generation number to be published for one
-of their products components. This avoids triggering an industry wide
+of their product's components. This avoids triggering an industry wide
 re-publishing of otherwise safe components.
 
- A product-specific minimum generation number only applies to the instance of
- that component that is signed with that product name. Another product's
- instance of the same component may be installed on the same system and would
- not be subject to the other product-specific minimum generation number.
- However, both of those components will need to meet the global minimum
- generation number for that component. A very likely scenario would be that a
- product is shipped with an incomplete fix required for a specific minimum
- generation number, but is labeled with that number. Rather than having the
- entire industry that uses that component re-release, just that product's
- minimum generation number is incremented and that product's component is
- re-released along with a UEFI variable update that specifies that requirement.
+ A product-specific minimum generation number only applies to the
+instance of that component that is signed with that product
+name. Another product's instance of the same component may be
+installed on the same system and would not be subject to the other
+product's product-specific minimum generation number. However, both of
+those components will need to meet the global minimum generation
+number for that component. A very likely scenario would be that a
+product is shipped with an incomplete fix required for a specific
+minimum generation number, but is labeled with that number. Rather
+than having the entire industry that uses that component re-release,
+just that product's minimum generation number would be incremented and
+that product's component re-released along with a UEFI variable update
+specifying that requirement.
```

Comment:
Why is this whole section getting re-flowed?

Reply:
apologies, editor damage :-(

Reply:
The original para is arnitrarily indented here, but meh - undoing this change for now and re-applying meaningful changes separately. :-)